Tests/L0/DeployTestAgent/_suite.ts

```diff
@@ -28,17 +28,17 @@ describe('Deploy Test Agent Suite', function () {
     });
 
     if (ps) {
-        it('(DeployTestAgent-ThrowIfPATNotExists) throws if no personal authentication token exists', (done) => {
-            psr.run(path.join(__dirname, 'ThrowIfPATNotExists.ps1'), done);
-        })
-        it('(DeployTestAgent-VerifyCompat.WindowsAgent) verifies if deploy test agent compat with windows agent', (done) => {
-            psr.run(path.join(__dirname, 'VerifyCompat.WindowsAgent.ps1'), done);
-        })
-        it('(DeployTestAgent-VerifyCompat.CoreCLRAgent) verifies if deploy test agent compat with coreclr agent', (done) => {
-            psr.run(path.join(__dirname, 'VerifyCompat.CoreCLRAgent.ps1'), done);
-        })
-        it('(DeployTestAgent-VerifyParameters) verifies if deploy test agent is called with the right parameters', (done) => {
-            psr.run(path.join(__dirname, 'VerifyParameters.ps1'), done);
-        })
+        // it('(DeployTestAgent-ThrowIfPATNotExists) throws if no personal authentication token exists', (done) => {
```

Comment:
What about files these tests are referring... ThrowIfPATNotExists.ps1, VerifyCompat.WindowsAgent.ps1, VerifyCompat.CoreCLRAgent.ps1, VerifyParameters.ps1

If not required those can also be removed.